uw-frame-components/css/angular.less

```diff
@@ -1,6 +1,6 @@
-@import "../bower_components/normalize-less/normalize.less";
-@import "../../bower_components/font-awesome/css/font-awesome.css";
-@import "../bower_components/bootstrap/less/bootstrap.less";
+@import "https://raw.githubusercontent.com/additiveinverse/normalize.less/master/normalize.less";
```

Comment:
Good catch - I tried to keep the same versions defined in bower, and in this case the version was '*'.  Should I just pick the latest tagged version?